README.md

```diff
@@ -120,7 +120,7 @@ available configuration parameters, see the [included example][5]).
     NCP:CCAThreshold              "-70"
 
 When up and running, `wpanctl` can be used to check the status of the
-interface and perform various management operations. For example, to
+interface and perform various management operations, for example, to
```

Comment:
Unfortunately, the second sentence isn't a sentence. I would prefer a comma, you prefer a hard stop, we can compromise on semi-colon if you are willing to use them. I prefer full sentences in writing, but again this is a personal preference. I should have taken a minimalist approach to your docs from the beginning. From now on, I will just flag egregious solecisms (= bad mistakes in grammar).